unlock-app/src/stories/creator/DemoPage.stories.js

```diff
@@ -41,7 +41,7 @@ const config = {
   providers: { HTTP: {}, Metamask: {} },
 }
 
-storiesOf('Demo', module)
+storiesOf('StaticDemo', module)
```

Comment:
I think it might be good to have a story both for the locked and unlocked states of the demo page, so we can see both forms of the paywall in context.

Reply:
But there was no such thing before that here? I want to make sure we have a single purpose PR?

Reply:
agreed. I think I made a PR for this exact thing a while ago, I have a branch sitting here locally which has that, it should be a follow-up